content/getting-started/white-label-apps.md

```diff
@@ -0,0 +1,95 @@
+---
+description: How to white label apps with Codemagic
+title: White label apps
+weight: 10
+---
+
+White labeling is the practice of building multiple versions of the same application. This can range from one or two up to thousands of different versions. 
+
+This documentation page should be considered as a minimum viable page to help answer most common questions developers have specifically about setting up CI/CD for their white label product in order to understand if they can achieve their desired use case with Codemagic.
+
+The documentation is open-sourced and we encourage you to provide feedback and contributions in the documentation GitHub repository [here](https://github.com/codemagic-ci-cd/codemagic-docs). If you are looking for general documentation about Codemagic, please use the search function or browse other parts of the documentation. 
+
+## How to get started with Codemagic when white labeling ~10 different versions
+
+We would recommend getting familiar with setting up a single version of your app to begin with. Set up your workflow using a codemagic.yaml configuration file and consult the documentation to understand core concepts such as using environment variables, build triggers, script steps, code signing and publishing. Once you have successfully built and published a version of you app to the stores, continue by adding addtional versions to your configuration file. 
```

Comment:
I suggest linking to https://docs.codemagic.io/yaml/yaml-getting-started/ here to for general info about yaml. 
```suggestion
We would recommend getting familiar with setting up a single version of your app to begin with. Set up your workflow using a codemagic.yaml configuration file and consult the documentation to understand core concepts, such as using environment variables, build triggers, script steps, code signing and publishing. Once you have successfully built and published a version of your app to the stores, continue by adding additional versions to your configuration file. 
```

Reply:
Added link.